amp/WEB-INF/src/org/digijava/module/aim/dbentity/ApprovalStatus.java

```diff
@@ -0,0 +1,56 @@
+package org.digijava.module.aim.dbentity;
+
+import org.digijava.module.aim.util.Identifiable;
+
+/**
+ *
+ * Change ids together with v_mondrian_activity_fixed_texts.xml, v_approval_status.xml, v_filtered_activity_status.xml.
+ *
+ * @author Octavian Ciubotaru
+ */
+public enum ApprovalStatus implements Identifiable {
+
+    /**
+     * In use. Not supported by reports. Should be removed in favor of started.
+     */
+    CREATED("created", 0),
+    APPROVED_STATUS("approved", 1),
+    EDITED_STATUS("edited", 2),
+    STARTED_APPROVED_STATUS("startedapproved", 3),
+    STARTED_STATUS("started", 4),
+    /**
+     * Not in use. Synonym to rejected?
+     */
+    NOT_APPROVED("not_approved", 5),
+    REJECTED_STATUS("rejected", 6);
```

Comment:
I was looking at `ApprovalStatus` to replicate in AMP Offline and decided to skip adding `_STATUS` suffix there. What do you think - should we catch the moment to simplify the name in AMP as well since the enum itself already explains that this about status?